### ISSUE TITLE
Alter the points for a driver in a particular race.

### ISSUE THREAD

**Issue by avs-code** [avs-code/PREM-Podium-race-E-manager#36]
Add a admin page to change points in DB, for example to 
> "enter a penalty for a driver (missed tracklimits, accidents or similar). If this would be possible, a text field to enter the reason for the modification is needed too". 
[http://manager.rackservice.org:50732/sessiondetails?sessionid=2539](url)
![captura](https://cloud.githubusercontent.com/assets/16170732/26783952/2897477a-49fb-11e7-8388-bc8e86d6d620.JPG)

**Comment by busterwolfeman** [comment]
After being more involved and got more knowledge about PREM I see that this is not possible with the actual structure of PREM. This would mean that the points have to be stored in the DB instead of calculated on the fly. However this would be much more intelligent, but it means lot of work.

**Comment by avs-code** [comment]
Yes, I know. it involves practically renewing how the system works. Several improvements or new features would imply that the system worked in another way. That may be to PREM 2.0 if it came to exist, because of the big changes that would have to be made.
